### DISCUSSION TITLE
Turbo support?

### DISCUSSION THREAD

**Discussion by ivodolenc** [sveltejs/kit#7432] [General]
Since Vercel anounce the new [Turbo](https://turbo.build/) system, what that means for SvelteKit?

Will SvelteKit add support for `turbo` or maybe migrate from `vite` to `turbo` completely?

I ask this because if this happens, it might be a good idea to do it before the first official release so there aren't too many breaking changes (some tips for community authors - plugin migration etc...).

**Comment by dummdidumm** [comment]
SvelteKit 1.0 will be based on Vite, no doubt about that! We are and will continue to be invested in the Vite ecosystem. One nice outcome of our refactoring efforts is that SvelteKit now consists of the core and a vite plugin. This means it's possible to also provide integration with Turbopack one day _in addition_ to Vite.

**Reply by ivodolenc** [reply]
Ok, thanks for the clarification.
